stream-chat-android-client/src/main/java/io/getstream/chat/android/client/utils/buffer/StartStopBuffer.kt

```diff
@@ -52,33 +60,33 @@ public class StartStopBuffer<T>(private val bufferLimit: Int = NO_LIMIT, customT
         active.set(false)
     }
 
-    public fun active() {
+    public fun active(src: String = "active") {
```

Comment:
Could we remove the default value?
It will ensure us we always provide a source tag to improve the debug
```suggestion
    public fun active(src: String) {
```